mapclassify/classifiers.py

```diff
@@ -1091,7 +1097,7 @@ def plot(
                 "Mapclassify.plot depends on matplotlib.pyplot, and this was"
                 "not able to be imported. \nInstall matplotlib to"
                 "plot spatial classifier."
-            )
+            ) from None
```

Comment:
Would using `except ImportError as err` and here 
```suggestion
            ) from err
```
make sense here and in other cases of the same pattern? Genuine question.

Reply:
My understanding is that `as err: .... from err` somewhat muddies the waters as to the traceback, but I could be mistaken. Thoughts?

Reply:
It can be nice in some cases but in the case of imports you're probably right.